axelor-sale/src/main/java/com/axelor/apps/sale/service/saleorder/SaleOrderLineServiceImpl.java

```diff
@@ -907,7 +917,8 @@ public void fillPriceFromPackLine(SaleOrderLine saleOrderLine, SaleOrder saleOrd
       inTaxPrice = fillDiscount(saleOrderLine, saleOrder, inTaxPrice);
       if (!saleOrderLine.getEnableFreezeFields()) {
         saleOrderLine.setPrice(
-            taxService.convertUnitPrice(true, saleOrderLine.getTaxLine(), inTaxPrice));
+            taxService.convertUnitPrice(
+                true, saleOrderLine.getTaxLine(), inTaxPrice, AppBaseService.COMPUTATION_SCALING));
```

Comment:
it looks like it is directly set in the `price` field in sale order line here, so please use the scale for unit price.